docs/_includes/footer.html

```diff
@@ -6,35 +6,31 @@
 			</div>
 			<div class="col-md-8 offset-md-1">
 				<div class="row">
-						<div class="col-md-4 col-xs-12">
-								<ul class="link-col">
-									<p class="footer-title"><a href="/docs/overview.html">Docs</a></p>
-									<li class="footer-resource"><a href="/docs/overview/why_secretless.html">Why Secretless?</a></li>
-									<li class="footer-resource"><a href="/docs/overview/how_it_works.html">How Does It Work?</a></li>
-									<li class="footer-resource"><a href="/docs/overview/key_terms.html">Key Terms</a></li>
-									<li class="footer-resource"><a href="/docs/reference.html">Reference</a></li>
-								</ul>
-							</div>
-
-							<div class="col-md-4 col-xs-12">
-								<ul class="link-col">
-									<p class="footer-title">Resources</p>
-									<li class="footer-resource"><a href="/community.html">Community</a></li>
-									<li class="footer-resource"><a href="/blog.html">Blog</a></li>
-									<li class="footer-resource"><a href="/docs/faq.html">FAQ</a></li>
-								</ul>
-							</div>
-
-							<div class="col-md-4 col-xs-12">
-								<ul class="link-col">
-										<p class="footer-title">Contribute</p>
-									<li class="footer-resource"><i class="fab fa-github"></i><a target="_blank" href="https://github.com/cyberark/secretless-broker" target="_blank">GitHub</a></li>
-									<li class="footer-resource"><i class="fas fa-comment-alt"></i><a target="_blank" href="https://groups.google.com/forum/#!forum/secretless" target="_blank">Google Group</a></li>
-								</ul>
-							</div>
+					<div class="col-md-4 col-xs-12">
+						<ul class="link-col">
+							<p class="footer-title"><a href="https://docs.secretless.io/">Docs</a></p>
+							<li class="footer-resource"><a href="/docs/overview/why_secretless.html">Why Secretless?</a></li>
```

Comment:
can we get the updated links from the new docs site and pre-load them here? it would be great if this PR could be ready to go for when it's time to flip the switch

Reply:
do we still want `references` to be in the footer? no references page is present in secretless staging. 
references in gh: https://github.com/cyberark/secretless-broker/blob/master/docs/docs/reference.md